src/styles/blocks/stars.css

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  gap: 4px;
```

Comment:
don't use gap, our test doesn't support it, use margin instead

Reply:
fixed